cob_script_server/src/simple_script_server/simple_script_server.py

```diff
@@ -1247,7 +1247,10 @@ def wait_for_finished(self, duration, logging):
 				if logging:
 					rospy.loginfo("Wait for <<%s>> reached <<%s>> (max %f secs)...",self.component_name, self.parameter_name,duration)
 				if not self.client.wait_for_result(rospy.Duration(duration)):
-					message = "Timeout while waiting for <<" + self.component_name +">> to reach <<" + self.parameter_name + ">>. Continuing..."
+					if type(self.parameter_name) is types.StringType:
+						message = "Timeout while waiting for <<" + self.component_name +">> to reach <<" + self.parameter_name + ">>. Continuing..."
+					else:
+						message = "Timeout while waiting for <<" + self.component_name +">>. Continuing..."
 					if logging:
```

Comment:
+1 for solution from @ipa-bnm, so we still have the information about the parameters content in the log